src/AxisSets.jl

```diff
@@ -1,5 +1,68 @@
 module AxisSets
 
+using AxisKeys
+using NamedDims
+
+"""
+    DEFAULT_FLATTEN_DELIM
+
+`:⁻` (or `\\^-`)
+
+Separates the parent symbols of a nested `NamedTuple` that has been flattened.
+A less common symbol was used to avoid collisions with `:_` in the parent symbols.
+
+# Example
+```jldoctest
+julia> using AxisSets: flatten
+
+julia> data = (
+           val1 = (a1 = 1, a2 = 2),
+           val2 = (b1 = 11, b2 = 22),
+           val3 = [111, 222],
+           val4 = 4.3,
+       );
+
+julia> flatten(data)
+(val1⁻a1 = 1, val1⁻a2 = 2, val2⁻b1 = 11, val2⁻b2 = 22, val3 = [111, 222], val4 = 4.3)
+```
+"""
+const DEFAULT_FLATTEN_DELIM = :⁻
+
+"""
+    DEFAULT_PROD_DELIM
+
+`:ᵡ` (or `\\^x`)
+
+Separates the parent symbols from an n-dimensional array that was flattened / reshaped.
```

Comment:
To distinguish the two kinds of operations. Technically, `ᵡ` has a stricter meaning because the resulting dimension is the product of the previous dimensions. The `⁻` is the more general form describing a nested relationship.

NOTE: I'll likely be including a 3rd symbol for `cat` operations

Reply:
I can make an issue to add a section in the docs about delimiter characters if you like?

Reply:
https://github.com/invenia/AxisSets.jl/issues/5

Reply:
>Technically, ᵡ has a stricter meaning because the resulting dimension is the product of the previous dimensions. 

Ah, I see. So `:objᵡloc` indicates we've taken the cross-product of all the elements in `:obj` and `:loc`, whereas `⁻` indicates we've flattened without necessarily taking a product.

>I can make an issue to add a section in the docs about delimiter characters if you like?

that would be helpful to have - thanks!